rke/cis_config.go

```diff
@@ -4,24 +4,75 @@ import (
 	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
 )
 
+const (
+	reasonNoConfigFile = `Cluster provisioned by RKE doesn't require or maintain a configuration file for kube-apiserver.
+All configuration is passed in as arguments at container run time.`
+	reasonForPSP       = `Enabling Pod Security Policy can cause issues with many helm chart installations`
+	reasonForNetPol    = `Enabling Network Policies can cause lot of unintended network traffic disruptions`
+	reasonForDefaultNS = `A default namespace provides a flexible workspace to try out various deployments`
+)
+
+var rkeCIS15NotApplicableChecks = map[string]string{
+	"1.1.1": reasonNoConfigFile,
+	"1.1.2": reasonNoConfigFile,
+	"1.1.3": reasonNoConfigFile,
+	"1.1.4": reasonNoConfigFile,
+	"1.1.5": reasonNoConfigFile,
+	"1.1.6": reasonNoConfigFile,
+	"1.1.7": reasonNoConfigFile,
+	"1.1.8": reasonNoConfigFile,
+}
+
+var rkeCIS15SkippedChecks = map[string]string{
```

Comment:
why are we adding these here and not system-chart?

Reply:
We just have one version of the system-chart, adding these there would require use of multiple versions.